tutorials/managing-gcp-projects-with-terraform/index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Managing GCP Projects with Terraform
+description: Learn how to manage projects within an organization and project resources with Terraform.
+author: danisla
+tags: Terraform
+date_published: --
+---
+
+This tutorial demonstrates how to create and manage projects on Google Cloud Platform with [Terraform](https://www.terraform.io/). Terraform is a tool for building, changing, and versioning infrastructure safely and efficiently by defining your infrastructure as code. Terraform supports multiple cloud platforms and software applications. If you are familiar with [Deployment Manager](https://cloud.google.com/deployment-manager/docs/), [Cloud Formation](https://aws.amazon.com/cloudformation/), or [Heat Templates](https://wiki.openstack.org/wiki/Heat) for Google Cloud Platform, Amazon Web Services and OpenStack respectively; Terraform is similar to these tools but not limited to any particular platform or provider. 
+ 
+Incremental changes made within your cloud project are difficult to track and recreate if needed later down the road. With Terraform, many of your resources like Projects, IAM policies, Networks, Compute Instances, and Container Engine Clusters can be managed and versioned and easily recreated for your organization or teams. The state that Terraform generates is saved to Google Cloud Storage (GCS) for persistence.
```

Comment:
Replace second and third sentences with: "With Terraform, many of your resources such as projects, IAM policies, networks, Compute Engine instances, and Container Engine clusters can be managed, versioned, and easily recreated for your organization or teams. The state that Terraform generates is saved to Google Cloud Storage for persistence."

Also, global: Do not abbreviate Google Cloud Storage as GCS. It's okay to just say "Cloud Storage" after the first reference, though.